### PR TITLE
Enhancement/skale 2929 Turn off reporting to skale manager

### PR DESCRIPTION
**Pull request by alexgex**

### CODE REVIEW

File: sla_agent.py [hunk 1]
```diff
@@ -91,6 +94,50 @@ def __init__(self, skale, node_id=None):
...
+        node_info = call_retry(self.skale.nodes.get, self.id)
+        return node_info['last_reward_date']
+
+    def generate_monitored_array(self):
+        active_ids = self.skale.nodes.get_active_node_ids()
+
+        active_ids.remove(self.id)
+
+        print(active_ids)
+        if len(active_ids) <= 24:
```
Comment (dmytrotkk) [inline sla_agent.py]: what is 24?
add a comment or move to constants

File: sla_agent.py [hunk 2]
```diff
@@ -91,6 +94,50 @@ def __init__(self, skale, node_id=None):
...
+    def get_last_reward_date(self):
+        node_info = call_retry(self.skale.nodes.get, self.id)
+        return node_info['last_reward_date']
+
+    def generate_monitored_array(self):
+        active_ids = self.skale.nodes.get_active_node_ids()
+
+        active_ids.remove(self.id)
+
+        print(active_ids)
```
Comment (dmytrotkk) [inline sla_agent.py]: debug print statement?
replace with logger or remove

File: sla_agent.py [hunk 3]
```diff
@@ -184,14 +231,15 @@ def monitor_job(self) -> None:
...
             skale = spawn_skale_lib(self.skale)
-            try:
-                self.nodes = call_retry.call(skale.monitors.get_checked_array, self.id)
-            except Exception as err:
-                self.logger.exception(f'Failed to get list of monitored nodes. Error: {err}')
-                self.notifier.send(f'Cannot save metrics to database - '
-                                   f'is MySQL container running? {err}', icon=MsgIcon.ERROR)
-                self.logger.info('Monitoring nodes from previous job list')
-
+            # try:
```
Comment (dmytrotkk) [inline sla_agent.py]: is this code temporarily commented or should be removed?

File: sla_agent.py [hunk 4]
```diff
@@ -231,15 +279,19 @@ def report_job(self) -> bool:
...
             monitor_w.jobqueue.put, self.monitor_job)
-        report_schedule = schedule.every(REPORT_PERIOD).minutes.do(
-            reporter_w.jobqueue.put, self.report_job)
         threading.Thread(target=monitor_w.worker, name='Monitor').start()
-        threading.Thread(target=reporter_w.worker, name='Reporter').start()
         monitor_schedule.run()
-        report_schedule.run()
+
+        # TODO: enable when move to validator-based monitoring
+        if False:
```
Comment (dmytrotkk) [inline sla_agent.py]: mb just comment out?

**Review by dmytrotkk — CHANGES_REQUESTED** [review]

File: sla_agent.py (hunk 1 above)
Comment (alexgex) [reply sla_agent.py]: these are temporarily added functions, but I moved it to constant anyway

File: sla_agent.py (hunk 2 above)
Comment (alexgex) [reply sla_agent.py]: removed

File: sla_agent.py (hunk 4 above)
Comment (alexgex) [reply sla_agent.py]: commenting will cause flake8 issues

File: sla_agent.py (hunk 3 above)
Comment (alexgex) [reply sla_agent.py]: temporarily commented

**Review by dmytrotkk — APPROVED** [review]
great!

**Pull request merged**